proposed/2021/NuGetConsumePDBInPackageReference.md

```diff
@@ -0,0 +1,216 @@
+# Consuming pdb support with PackageReference
+
+* Status: Reviewing
+* Author(s): [Heng Liu](https://github.com/heng-liu)
+* Issue: [5926](https://github.com/NuGet/Home/issues/5926) Consume pdbs from packages in PackageReference - add Debug Symbols to LockFileTargetLibrary
+* Type: Feature
+
+## Problem Background
+
+In the PackageReference world, the packages are consumed from the global-packages folder, .pdb, .xml files from lib and runtime folder are not copied into the build output folder.
```

Comment:
PDB and XML documentation files are the most common related files, but RAR can find arbitrary related files. The list of possible extensions is user-configurable and the default is here: https://github.com/dotnet/msbuild/blob/2f1e9cad51097aec5d7268a4afdf7ad9bc6864b9/src/Tasks/Microsoft.Common.CurrentVersion.targets#L617-L627

It looks like the proposal covers this; the only thing is we'll have to be careful about "differs only by extension" to cover cases like `.dll.config`.

Reply:
Thanks for reminding me for that!
So how about adding the following as additive description in the doc? (Haven't added yet)
The assembly extension `.winmd`, `.dll` and `.exe` will not be listed in "related" property. Any other extensions, like `.dll.config`, will be listed in "related" property.